.github/workflows/cd.yml

```diff
@@ -0,0 +1,19 @@
+name: ALIAS-SOLVD-TEAM2 Node.js CD
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
why two empty lines above?

Reply:
That's a very sharp point of attention, thank you